lib/ex_unit/lib/ex_unit/case.ex

```diff
@@ -534,11 +534,12 @@ defmodule ExUnit.Case do
     {name, describe, describe_line, describetag} =
       case Module.get_attribute(mod, :ex_unit_describe) do
         {line, describe, _counter} ->
-          description = :"#{test_type} #{describe} #{name}"
-          {description, describe, line, Module.get_attribute(mod, :describetag)}
+          description = validate_name("#{test_type} #{describe} #{name}")
+          {:"#{description}", describe, line, Module.get_attribute(mod, :describetag)}
 
         nil ->
-          {:"#{test_type} #{name}", nil, nil, []}
+          description = validate_name("#{test_type} #{name}")
+          {:"#{description}", nil, nil, []}
```

Comment:
According to the suggestion below:

```suggestion
          test_name = validate_test_name("#{test_type} #{name}")
          {test_name, nil, nil, []}
```